main/EDTypesMod.F90

```diff
@@ -518,6 +519,43 @@ end subroutine ZeroMassBalFlux
    
   ! =====================================================================================
 
+  subroutine CalculateTreeGrassArea(csite, tree_fraction, grass_fraction, bare_fraction)
```

Comment:
It took me a while to see how this differs from `UpdateTreeGrassArea`. One suggestion may be to rename this to `UpdateTreeGrassAreaSite` and the other routine to `UpdateTreeGrassAreaPatch`, just so the difference is clearer from their names (although longer names).

Reply:
As discussed in the FATES meeting, because the patch method will always be called with patch%UpdateTreeGrassArea I am going to leave that one and only change the site one as you suggested.